shardingsphere-governance/shardingsphere-governance-core/shardingsphere-governance-core-common/src/main/java/org/apache/shardingsphere/governance/core/yaml/config/metadata/YamlRuleSchemaMetaData.java

```diff
@@ -32,5 +33,5 @@
 
     private YamlSchemaMetaData configuredSchemaMetaData;
 
-    private Map<String, YamlSchemaMetaData> unconfiguredSchemaMetaDataMap;
+    private Map<String, Collection<String>> unconfiguredSchemaMetaDataMap;
```

Comment:
This PR looks great. However, I found another issue relevant to what you modified. 

```java
// CreateTableStatementMetaDataRefreshStrategy

private void refreshUnconfiguredMetaData(final ShardingSphereMetaData metaData, 
                                             final DatabaseType databaseType, final Map<String, DataSource> dataSourceMap, final String tableName) throws SQLException {
        for (Entry<String, DataSource> entry : dataSourceMap.entrySet()) {
            Optional<TableMetaData> tableMetaData = TableMetaDataLoader.loadWithoutColumnMetaData(entry.getValue(), tableName, databaseType.getName());
            if (tableMetaData.isPresent()) {
                refreshUnconfiguredMetaData(metaData, tableName, entry.getKey());
                return;
            }
        }
    }

```
`loadWithoutColumnMetaData` is supposed to be simplified, IMO. What do you think?

Reply:
done